### PR TITLE
narrative for new units of measure and magnitudes

### PR DESCRIPTION
**Pull request by philblackwood**
Narrative describing new magnitudes and units of measure. Closes issue #1035

### CODE REVIEW

File: docs/release_notes/issue1035-uom-narrative.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+The following is an informal high-level description of changes to gist to better support magnitudes and units of measure.
+
+With gist 13, the ontology for magnitudes and units of measure has been revisited to provide:
+
+- Simpler governance
+- Support for financial metrics
+- Simple standard ways to represent data involving units of measure
+- Support for calculations involving units of measure
+
+The release includes scripts for converting data from gist 12 to gist 13, and Semantic Arts will also provide a rich set of reference data for units of measure that can be tapped going forward.
```
Comment (uscholdm) [inline docs/release_notes/issue1035-uom-narrative.md:14]: Please include something about what they are supposed to do in the meantime, given we have removed all of the units and conversion.

File: docs/release_notes/issue1035-uom-narrative.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+- has unit of measure
+- numericValue
+
+## Main concepts
+
+Building on this example, the new concepts for units of measure in gist 13 are:
+
+- every magnitude is directly related to a measurable characteristic (called an aspect)
+- aspects and units of measure are treated as reference data and are not part of the ontology
+- groups of units can be established, where all the units in the group measure the same aspect (for example bit, kilobit, and megabit all measure a quantity of data)
```
Comment (uscholdm) [inline docs/release_notes/issue1035-uom-narrative.md:42]: I sugguest using a more common unit in the example. E.g. 
REplace
- `(for example bit, kilobit, and megabit all measure a quantity of data)`
with 
- `(for example foot, meter and kilometer all measure distance)`

File: docs/release_notes/issue1035-uom-narrative.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+- the units of measure in a group are all multiples of the same product of powers of base units, given as exponent of ampere, exponent of bit, exponent of candela, etc.
```
Comment (uscholdm) [inline docs/release_notes/issue1035-uom-narrative.md:43]:
```suggestion
- the units of measure in a group are all multiples of the same product of powers of base units, given as exponent of meter, exponent of second, exponent of ampere, etc.

File: docs/release_notes/issue1035-uom-narrative.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+- aspects can be categorized by discipline
+
+## What's new and different
+
+To draw a comparison with gist 12:
+
+- it is no longer necessary to change the ontology to add new aspects and units of measure, because they are treated as reference data instead of as classes
+- it is no longer necessary to figure out if a unit is a coherent unit, a simple unit, a product unit, etc.
+- because of the two items above, gist now has 25% fewer classes
+- there is now one standard way to represent things that previously could be represented multiple ways
```
Comment (uscholdm) [inline docs/release_notes/issue1035-uom-narrative.md:55]: Different points:
- They still can be represented in multiple ways, what we are doing is providing an easier approach that allows use of a single way. 
- SA has gone to great lengths to illustrate this, and many of us understand it, but its not at all easy to grok.  Maybe a brief summary here will help with a pointer to the documentation in the migration scripts which spells out these differences.

File: docs/release_notes/issue1035-uom-narrative.md [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+- there is now a simple way to validate that a unit of measure is applicable for measuring a given aspect
```
Comment (uscholdm) [inline docs/release_notes/issue1035-uom-narrative.md:56]: Can you give a hint as to what this is?

File: docs/release_notes/issue1035-uom-narrative.md [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+- calculations can now be done in SPARQL that would be far more complex in gist 12
```
Comment (uscholdm) [inline docs/release_notes/issue1035-uom-narrative.md:57]: For example?

File: docs/release_notes/issue1035-uom-narrative.md [hunk 7]
```diff
@@ -0,0 +1,131 @@
...
+
+In terms of the ontology:
+
+- the conversion factor for watt-hour is 3600
+- the exponent of kilogram is 1
+- the exponent of meter is 2
+- the exponent of second is -2
+- all other exponents are zero
+
+Every member of a unit group containing watt-hour must be a multiple of kilogram meter squared per second squared. The unit group containing watt-hour would also contain kilowatt-hour, for example.
```
Comment (uscholdm) [inline docs/release_notes/issue1035-uom-narrative.md:73]:
```suggestion
Every member of a unit group containing watt-hour must be a multiple of kilogram meter squared per second squared. The unit group containing watt-hour might also contain kilowatt-hour, for example.

File: docs/release_notes/issue1035-uom-narrative.md [hunk 8]
```diff
@@ -0,0 +1,131 @@
...
+
+## Working with exponents
+
+Reminder: calculations involving the exponents can be done as follows:
```
Comment (uscholdm) [inline docs/release_notes/issue1035-uom-narrative.md:77]: I don't think its a reminder

```suggestion
Calculations involving the exponents can be done as follows:

File: docs/release_notes/issue1035-uom-narrative.md (hunk 7 above)
Comment (kchastain703) [reply docs/release_notes/issue1035-uom-narrative.md:73]: I think "would" works here in the sense that, conceptually, kilowatt-hour would be in the same unit group whether the user is defining/using it or not. And if they are using kilowatt-hour, it's a definite, concrete example of the unit group it does belong to.

File: docs/release_notes/issue1035-uom-narrative.md (hunk 1 above)
Comment (philblackwood) [reply docs/release_notes/issue1035-uom-narrative.md]: The description of the conversion process talks about how an Enterprise can get a standard set of units and aspects from the Semantic Arts reference data.

File: docs/release_notes/issue1035-uom-narrative.md [hunk 9]
```diff
@@ -0,0 +1,131 @@
...
+With gist 13, the ontology for magnitudes and units of measure has been revisited to provide:
+
+- Simpler governance
+- Support for financial metrics
+- Simple standard ways to represent data involving units of measure
+- Support for calculations involving units of measure
+
+The release includes scripts for converting data from gist 12 to gist 13, and Semantic Arts will also provide a rich set of reference data for units of measure that can be tapped going forward.
+
+## Simple example
```
[the remark this answers is not in the document]
Comment (philblackwood) [reply docs/release_notes/issue1035-uom-narrative.md]: The information I could find about this talks about creating a separate repository for images and then using relative path names. The discussions get pretty involved about pros and cons, so until we adopt a standard practice for doing this I am not confident I know the implications.

File: docs/release_notes/issue1035-uom-narrative.md (hunk 2 above)
Comment (philblackwood) [reply docs/release_notes/issue1035-uom-narrative.md]: done

File: docs/release_notes/issue1035-uom-narrative.md (hunk 3 above)
Comment (philblackwood) [reply docs/release_notes/issue1035-uom-narrative.md]: done

File: docs/release_notes/issue1035-uom-narrative.md (hunk 4 above)
Comment (philblackwood) [reply docs/release_notes/issue1035-uom-narrative.md]: done

File: docs/release_notes/issue1035-uom-narrative.md (hunk 5 above)
Comment (philblackwood) [reply docs/release_notes/issue1035-uom-narrative.md]: done

File: docs/release_notes/issue1035-uom-narrative.md (hunk 6 above)
Comment (philblackwood) [reply docs/release_notes/issue1035-uom-narrative.md]: the examples are in the following sections

File: docs/release_notes/issue1035-uom-narrative.md (hunk 7 above)
Comment (philblackwood) [reply docs/release_notes/issue1035-uom-narrative.md]: omitted the reference to kilowatt-hours.

File: docs/release_notes/issue1035-uom-narrative.md (hunk 8 above)
Comment (philblackwood) [reply docs/release_notes/issue1035-uom-narrative.md]: removed "Reminder"

File: docs/release_notes/issue1035-uom-narrative.md (hunk 1 above)
Comment (uscholdm) [reply docs/release_notes/issue1035-uom-narrative.md]: Yes, but noone has access to that data at this time. What do we recommend they do for the time being?

File: docs/release_notes/issue1035-uom-narrative.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (uscholdm) [reply docs/release_notes/issue1035-uom-narrative.md]: Yeah, they way github does all that magic behind the scenes is remarkable. We'll drop this.

File: docs/release_notes/issue1035-uom-narrative.md (hunk 6 above)
Comment (uscholdm) [reply docs/release_notes/issue1035-uom-narrative.md]: ok

File: docs/release_notes/issue1035-uom-narrative.md (hunk 7 above)
Comment (uscholdm) [reply docs/release_notes/issue1035-uom-narrative.md]: ok

File: docs/release_notes/issue1035-uom-narrative.md (hunk 5 above)
Comment (uscholdm) [reply docs/release_notes/issue1035-uom-narrative.md]: I just looked at the commit changes and did not see an added hint. Can you point it out?

**Review by rjyounes — CHANGES_REQUESTED** [review]

**Review by rjyounes — APPROVED** [review]

**Pull request merged**